src/Microsoft.DotNet.Cli.Utils/Muxer.cs

```diff
@@ -11,6 +11,14 @@ public class Muxer
 
         private string _muxerPath;
 
+        internal string SharedFxVersion
+        {
+            get
+            {
+                return new DirectoryInfo(Path.GetDirectoryName(GetDataFromAppDomain("FX_DEPS_FILE"))).Name;
```

Comment:
I'd prefer:
```
   var depsFile = new FileInfo(GetDataFromAppDomain("FX_DEPS_FILE"));
   return depsFile.Directory.Name;
```

That might not compile :tm: but you get the idea. It's the name of the directory that contains the deps file. That'd be easier to parse than what we have right now...